src/server/auth/cmds/cmds.go

```diff
@@ -243,6 +243,11 @@ func DeactivateCmd() *cobra.Command {
 				return errors.Wrapf(err, "could not connect")
 			}
 			defer c.Close()
+
+			// Delete any data from the identity server
+			if _, err := c.IdentityAPIClient.DeleteAll(c.Ctx(), &identity.DeleteAllRequest{}); err != nil {
```

Comment:
Maybe this is too much of a corner case to worry about, but is there any risk that by doing `identity.DeleteAll()` before `auth.Deactivate()`, we could get the user's cluster stuck? I.e. they call `c.Deactivate()`, but because that call is restricted they need to re-auth, but they can no longer re-auth because the OIDC info is gone?

Reply:
I think this corner case is covered by (a) the root token (b) the ability to do postgres surgery to create a new root token if the original is lost.